pkg/version/version.go

```diff
@@ -17,7 +17,7 @@
 package version
 
 // Terrascan The Terrascan version
-const Terrascan = "v1.0.0"
+const Terrascan = "v1.0.1"
```

Comment:
Let's make this v1.1.0 since we're adding significant functionality and not just bug fixes.

Reply:
yep will do